transforms/code/proglang_select/ray/src/proglang_select_transform_ray.py

```diff
@@ -48,6 +46,8 @@
 
 
 def _get_supported_languages(lang_file: str, data_access: DataAccess) -> list[str]:
+    from data_processing.utils import get_logger
+    logger = get_logger(__name__)
```

Comment:
maybe the logger should be passed in as an argument.  Otherwise, might this be different than what the transform has?

Reply:
good idea